### PR TITLE
Postmanager

### PR DESCRIPTION
**Pull request by FXLaroche**
User Story: create a new post. Completed. 

This story will need to be reopened once the session has been coded to integrate the user_id into the form from the $_SESSION variable.

### CODE REVIEW

File: src/Controller/ItemController.php [hunk 1]
```diff
@@ -23,6 +23,7 @@ public function index(): string
      */
     public function show(int $id): string
     {
+        var_dump($_GET);
```
Comment (aldaitzwild) [inline src/Controller/ItemController.php:26]: No you don't push a Var_dump() on dev

File: src/Controller/PostController.php [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+                $error[] = "Please put a message, it's better.";
+            }
+
+            if (strlen($newPost['keyword']) > 255) {
+                $error[] = "Alas! you have too many keywords!";
+            }
+
+            /*if ($newPost['user_id'] != $_SESSION['user_id'])
+                $error[] = 'Who do you think you are?!';*/
+            //set as commentary, needs to be unlocked once the Session part has been coded.
```
Comment (aldaitzwild) [inline src/Controller/PostController.php:42]: No dead code (commented code ) in the PR.

**Review by aldaitzwild — CHANGES_REQUESTED** [review]
You also have conflict with the dev branch.
So you first need to merge dev into your branch, and then push it for a PR

**Comment by FXLaroche** [comment]
Modifications done.

File: src/View/layout.html.twig [hunk 3]
```diff
@@ -1,12 +1,18 @@
...
-    <meta name="viewport" content="width=device-width, initial-scale=1">
+	<head>
+		<meta charset="utf-8">
+		<meta http-equiv="x-ua-compatible" content="ie=edge">
+		<meta name="viewport" content="width=device-width, initial-scale=1">
+		<link href="https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-1BmE4kWBq78iYhFldvKuhfTAU6auU8tT94WrHftjDbrCEXSU1oBoqyl2QvZ6jIW3" crossorigin="anonymous">
 
-    <title>{% block title %}{% endblock %}</title>
 
+<<<<<<< HEAD
```
Comment (aldaitzwild) [inline src/View/layout.html.twig:10]: So merging branches is great, BUT resolving conflicts is GREATER !!!! You don't push a branch where conflicts still exists

**Review by aldaitzwild — CHANGES_REQUESTED** [review]

**Review by aldaitzwild — CHANGES_REQUESTED** [review]
New conflicts has poped as dev progessed : 
Conflicting files
.gitignore
src/Controller/PostController.php
src/Model/PostManager.php
src/View/layout.html.twig
src/routes.php


You need to merge it again :-)

**Review by aldaitzwild — APPROVED** [review]

**Pull request merged**